schema/me/conversations.js

```diff
@@ -10,11 +10,11 @@ export default {
   args: pageable(),
   resolve: (root, options, request, { rootValue: { conversationsLoader } }) => {
     if (!conversationsLoader) return null
-    const relayOptions = parseRelayOptions(options)
-    return conversationsLoader(relayOptions).then(({ conversations }) => {
+    const { page, size, offset } = parseRelayOptions(options)
+    return conversationsLoader({ page, size }).then(({ total_count, conversations }) => {
```

Comment:
We only want to pass in `page` and `size` to the Impulse API, and then we need to use the `total_count` (for Relay to know there are subsequent pages).